rrrspec-client/lib/rrrspec/client/slave_runner.rb

```diff
@@ -57,9 +59,12 @@ def work
           @timeout = TASKQUEUE_TASK_TIMEOUT
           trial = Trial.create(task, @slave)
 
-          @rspec_runner.reset
+          task_file_ext = File.extname(task.spec_file)
+          runner = RunnerFactory.create(task_file_ext)
```

Comment:
BUG: RSpecRunner has a before_suite_run flag for oneshot suite-wide initialization. New-ing the runner for each task resets this flag and causes multiple initialization.